src/super_gradients/training/models/segmentation_models/unet/unet.py

```diff
@@ -214,3 +214,15 @@ def __init__(self, arch_params: HpmStruct):
             aux_heads_params=get_param(arch_params, "aux_heads_params"),
             dropout=get_param(arch_params, "dropout", 0.0),
         )
+
+
+class UNet(UNetCustom):
+    """
+    implementation of:
+     "U-Net: Convolutional Networks for Biomedical Image Segmentation", https://arxiv.org/pdf/1505.04597.pdf
+    The upsample operation is done by using bilinear interpolation which is reported to show better results.
```

Comment:
If we want to support the original Conv transpose version, we'll have to implement a custom up block only for the vanilla UNet.
The original conv transpose use kernel=2 and stride=2 (not recommended since there is no spatial dependencies with adjacent pixels), and out_channels == in_channels // 2